src/main.cc

```diff
@@ -9,8 +9,12 @@ int main(int argc, char* argv[]) {
         return 1;
     }
 
-    Options opt;
-    short port_num = opt.getPort(argv[1]);
+    Options opt(argv[1]);
+    short port_num = opt.getPort();
+    if (port_num == -1) {
+        std::cerr << "Was not able to get port number from <config_file>.\n";
+        return 1;
+    }
```

Comment:
For future options, we might want to find a better way of signaling parsing error. Maybe we can move this error reporting into `Options` itself? Let's do that later, in any case.